### ISSUE TITLE
Name changes in 2.4 open beta [Spoiler]

### ISSUE THREAD

**Issue by CMDR-Cometborne** [EDDiscovery/EDDiscovery#1229]
The "unknown" materials, data, and cargo have changed their names in 2.4 beta.  The names in the journal are unchanged, only the display in games changes:

Type | Journal Name | Name in game in 2.3 | Name in game in 2.4
-- | -- | -- | --
Material | unknowncarapace | Unknown Carapace | Thargoid Carapace
Material | unknownenergysource | Unknown Fragment | Sensor Fragment
Material | unknownenergycell | Unknown Energy Cell | Thargoid Energy Cell
Material | unknownorganiccircuitry | Unknown Organic Circuitry | Thargoid Organic Circuitry
Material | unknowntechnologycomponents | Unknown Technology Components | Thargoid Technological Components
Data | tg_compositiondata | Unknown Material Composition Data | Thargoid Material Composition Data
Data | tg_residuedata | Unknown Residue Data Analysis | Thargoid Residue Data
Data | unknownshipsignature | Unknown Ship Signature | Thargoid Ship Signature
Data | tg_structuraldata | Unknown Structural Data | Thargoid Structural Data
Data | unknownwakedata | Unknown Wake Data | Thargoid Wake Data
Cargo | unknownartifact | Unknown Artefact | Thargoid Sensor
Cargo | unknownartifact2 | Unknown Probe | Thargoid Probe
Cargo | unknownartifact3 | Unknown Link | Thargoid Link
Cargo | unknownresin | Unknown Resin | Thargoid Resin
Cargo | unknownbiologicalmatter | Unknown Biological Matter | Thargoid Biological Matter
Cargo | unknowntechnologysamples | Unknown Technology Samples | Thargoid Technology Samples

In summary, most is 1,$s/Unknown/Thargoid/g, except for the change from Unknown Artefact to Thargoid Sensor, Unknown Fragment to Sensor Fragment, and Technology Components to Technological Components.

**Comment by phroggster** [comment]
Closing this issue, as it has been resolved in v8.3.1 via c0b00204da0dd2e72344364890c83118c284fd3e.
